scipy/optimize/tests/test_optimize.py

```diff
@@ -913,5 +913,49 @@ def test_attributes_present(self):
                 assert_(hasattr(res, attribute))
 
 
+class TestBrute:
+    # Test the "brute force" method
+    def setUp(self):
+        self.params = (2, 3, 7, 8, 9, 10, 44, -1, 2, 26, 1, -2, 0.5)
+        self.rranges = (slice(-4, 4, 0.25), slice(-4, 4, 0.25))
+        self.solution = np.array([-1.05665192, 1.80834843])
+
+    def f1(self, z, *params):
+        x, y = z
+        a, b, c, d, e, f, g, h, i, j, k, l, scale = params
+        return (a * x**2 + b * x * y + c * y**2 + d*x + e*y + f)
+
+    def f2(self, z, *params):
+        x, y = z
+        a, b, c, d, e, f, g, h, i, j, k, l, scale = params
+        return (-g*np.exp(-((x-h)**2 + (y-i)**2) / scale))
+
+    def f3(self, z, *params):
+        x, y = z
+        a, b, c, d, e, f, g, h, i, j, k, l, scale = params
+        return (-j*np.exp(-((x-k)**2 + (y-l)**2) / scale))
+
+    def func(self, z, *params):
+        x, y = z
+        a, b, c, d, e, f, g, h, i, j, k, l, scale = params
```

Comment:
Oh, you're right! I copy-pasted that test from the docs (Examples) of optimize.brute. I'll remove the lines from the docs as well.